source/features/update-pr-from-base-branch.tsx

```diff
@@ -28,38 +28,43 @@ async function mergeBranches(): Promise<AnyObject> {
 	});
 }
 
-async function handler(event: delegate.Event): Promise<void> {
-	const button = event.target as HTMLButtonElement;
-	button.disabled = true;
-	button.textContent = 'Updating branch…';
-	button.classList.remove('tooltipped');
+async function handler({delegateTarget}: delegate.Event): Promise<void> {
+	if (!confirm(delegateTarget.getAttribute('aria-label')! + '?')) {
+		return;
+	}
+
+	const statusMeta = delegateTarget.parentElement!;
+	statusMeta.textContent = 'Updating branch…';
 	observer.disconnect();
 
 	const response = await mergeBranches();
 	if (response.ok) {
-		button.remove();
+		statusMeta.remove();
 	} else if (response.message?.toLowerCase().startsWith('merge conflict')) {
 		// Only shown on Draft PRs
-		button.replaceWith(
+		statusMeta.textContent = '';
+		statusMeta.append(
 			<a href={location.pathname + '/conflicts'} className="btn float-right"><AlertIcon/> Resolve conflicts</a>
 		);
 	} else {
-		button.textContent = response.message ?? 'Error';
-		button.prepend(<AlertIcon/>, ' ');
+		statusMeta.textContent = response.message ?? 'Error';
+		statusMeta.prepend(<AlertIcon/>, ' ');
 		throw new api.RefinedGitHubAPIError('update-pr-from-base-branch: ' + JSON.stringify(response));
 	}
 }
 
 function createButton(base: string, head: string): HTMLElement {
-	return (
-		<button type="button" className="btn float-right rgh-update-pr-from-master tooltipped tooltipped-n" aria-label={`Merge the ${base} branch into ${head}`}>
-			Update branch
+	const button = (
+		<button type="button" className="btn-link tooltipped tooltipped-n" aria-label={`Merge the ${base} branch into ${head}`}>
```

Comment:
Since we ask for confirmation now, this seems duplicate/unnecessary
```suggestion
		<button type="button" className="btn-link">
```

Reply:
Done.